src/pages/User/components/UserHeader/UserHeader.vue

```diff
@@ -31,9 +31,7 @@
             <div class="UserHeader__text">
               <p
                 :class="`UserHeader__membership ${
-                  isMembershipActive
-                    ? 'UserHeader__membership--active'
-                    : 'UserHeader__membership--inactive'
+                  isMembershipActive || 'UserHeader__membership--inactive'
```

Comment:
I don't think this is good it will be "false" in some cases, and that is not a valid CSS class

Reply:
restore after deeming it necessary to highlight the state with colors